src/app.js

```diff
@@ -1,21 +1,23 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const fs = require('fs');
+const url = require('url');
 
-module.exports = sum;
+const PORT = process.env.PORT || 3006;
+
+const server = http.createServer((req, res) => {
+  const normalizedUrl = new url.URL(req.url, `http://${req.headers.host}`);
+  const fileName = normalizedUrl.pathname.slice(1) || 'index.html';
+
+  fs.readFile(`.file/${fileName}`, (err, data) => {
```

Comment:
you need to read files from public 
![image](https://github.com/mate-academy/node_static-files-server/assets/96265844/b9676358-0156-441b-8138-17923a99ee47)

Reply:
and you need to check if path have file inside and then cut it out